doc/tasks/readme.md

```diff
@@ -0,0 +1,737 @@
+markiert werden. Es dürfen keine falschen Antworten markiert werden. Die Reihenfolge spielt
+keine Rolle.
+
+### Lückentext
+
+Die Aufgabenstellung besteht aus einem Fragetext. Die Antwort besteht aus einem oder
+mehreren Freitexten. Es muss jeweils die richtige Antwort in den Freitext-Stellen eingegeben
+werden. Die richtige Antwort kann ein konkreter Wert oder eine Eingabe passend zu einem
+regulären Ausdruck sein.
+
+### Ersetzen
+
+Aus einer Gesamtmenge aus $l$ Elementen (mit $0 < l$) muss eine Menge mit $m$ Elementen (mit $0 < n$) durch eine
+zweite Menge mit $m$ Elementen (mit $0 < m$) ersetzt werden.
+Hierdurch wird die Gesamtmenge der Elemente sowie die Anzahl der enthaltenen Elemente verändert.
+
+### Zuordnen
+
+Zuordnen bezeichnet im Allgemeinen die Aufgabe, Elemente aus einer Menge $A$ (z.B.
+Antwortmöglichkeiten) die Elemente aus einer anderen Menge $B$ zuzuordnen (bspw. Array-Index,
+Reihe-Spalte Kombination einer Matrix). Hierbei muss nicht jedes Element aus $A$ der
+Antwortmöglichkeiten auch einem Element aus $B$ zugeordnet werden. Allerdings muss jedem Element aus $B$ ein
+Element aus $A$ zugeordnet werden. Die gesamte Menge $A$ der Antwortmöglichkeiten muss den Studierenden
+bekannt sein, bevor sie den ersten Zuordnungsschritt machen müssen.
+
+### Elemente in der richtigen Reihenfolge auswählen
+
+Es muss in der richtigen Reihenfolge ein Element nach dem anderen aus einer Menge $A$ ausgewählt
+werden.
+Den Studierenden müssen nicht alle Elemente aus $A$ bekannt sein, bevor sie das erste Element
+auswählen müssen.
+
+## Konkrete Entitätstypen
+
+- Questgeber: Entität, über die eine Aufgabe aktiviert wird
+- Bild: Interaktive Entität im Dungeon, die bei Interaktion ein Bild anzeigt (bspw. ein UML-Diagram, Graph, etc.)
+- Container: Entität, deren Inventar vom Spielcharakter manipuliert werden kann
+- Monster: Entität, welche den Spielcharakter (nach einem bestimmten Verhalten) verfolgt, angreift und besiegt werden kann
+- interaktives Bedienelement: Eine interaktive Entität, deren Zustand vom Spielcharakter manipuliert werden kann. Es sind vier
+  unterschiedliche Arten vorgesehen:
+  - Tastend: Das Bedienelement kann impulsartig betätigt werden, d.h. bei Interaktion wird einmalig eine Aktion ausgeführt
+  - Schaltend: Das Bedienelement kann zwischen zwei Zuständen getoggelt werden (an/aus)
+  - Zustandsmenge > 2: Das Bedienelement kann zwischen mehr als 2 fest definierten Zuständen umgeschaltet werden
+  - Kontinuierlich: Das Bedienelement verhält sich wie ein Schieberegler, bei dem ein Wert zwischen einer oberen und einer unteren
+    Grenze eingestellt werden kann
+- Item: Eine Entität, die entweder eine Position im Dungeon haben kann (also bspw. auf dem Boden liegt), oder in einem
+  Inventar gespeichert sein kann und zwischen diesen beiden Zuständen wechseln kann (das ist bewusst vereinfacht, ohne
+  auf die Feinheiten der Item=Entität Thematik einzugehen)
```

Comment:
Den Nachsatz bitte streichen - das versteht man in diesem Zusammenhang nicht. Oder auf die Diskussion im Game verlinken (Doku und/oder Issue).